src/components/varsom-avalanche-activity-obs/varsom-avalanche-activity-obs.tsx

```diff
@@ -33,6 +34,10 @@ export class VarsomAvalancheActivityObs {
   @Prop() Comment: any;
   @Prop() Attachments: Attachment[];
   
+  async componentWillLoad(){
```

Comment:
Dette bryter med https://wiki.c2.com/?OnceAndOnlyOnce
I stedet for å ha nøyaktig den samme funksjonen i alle disse klassene, kunne du antakelig løst dette med å ha en felles baseklasse for disse komponentene.

Reply:
Det var synd. Men kanskje det er like greit. Om du får til å laste tekster i render-metoden, holder det at tekst-objektet er en lokal variabel i render-metoden. Da slipper vi unna med en ekstra linje i hver komponent:
`const strings = await getLocaleComponentStrings(getLocaleFromDom());`
eller enda enklere:
`const strings = await getLocaleComponentStrings();`
hvis du lar lang-parameteren i getLocaleComponentStrings() være valgfri og heller kalle getLocaleFromDom() i funksjonen getLocaleComponentStrings() hvis lang ikke er satt.